src/compute-pr-actions.ts

```diff
@@ -237,7 +231,6 @@ export function process(prInfo: BotResult,
             return {
                 ...createEmptyActions(),
                 targetColumn: "Needs Author Action",
-                shouldUpdateProjectColumn: true
```

Comment:
Picky: this leaves a final `,` in this object, so either drop it or do that for the other one?  (Maybe for all multi-line objects?)

Reply:
Done :heavy_check_mark: #347